CHANGELOG.md

```diff
@@ -5,7 +5,8 @@
 * [ENHANCEMENT] ...
 * [BUGFIX] ...
 
-## 0.29.0-rc.1 / 2025-10-122
+## 0.29.0-rc.2 / 2025-10-31
+* [BUGFIX] Fix Jira issue count #4615
 
 ## 0.29.0-rc.0 / 2025-10-09
```

Comment:
We don't add new versions for RCs, we just replace the previous version and update the changelog.